docs/technical_docs/Inji_Verify_API_Overview.md

```
@@ -278,6 +290,43 @@ The `response_code` is short-lived, single-use, and cryptographically secure.
 
 ---
 
+### POST /vp-submission/dc-api
+
+Browser SDK submits the Verifiable Presentation for Digital Credentials API sessions (`response_mode=dc_api`). Accepts `application/json`. Called by the verifier SDK, not the wallet.
+
+**Request body**
+```json
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)